Write up  process of nominating maintainer (#206)
astropy/astropy-project: policies/adding-new-maintainers.md

> This document describes the process for adding new people to maintainer roles.
> "Maintainer" here basically means any role with github write powers. The

Reviewer: ```suggestion
Maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). Therefore, a maintainer usually has experience with a package e.g. through work they did themselves or by commenting and helping others to work on PRs in a package or sub-package. There is no formal minimum on the number of commits/PRs/etc. though, anyone can volunteer or be suggested as a maintainer following the process below.

This document describes the process for adding new people to maintainer roles.
"Maintainer" here basically means any role with github write powers. The
```

Reviewer: We don't have the mentoring program yet, but this would be a good place to give a shout out about that.

Author: (I'm accepting this suggestion, but am going to modify it a bit further in a commit to push up)

process is in the hands of the Coordination Committee according to

> [APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst), which also
> requires the comment period. The process is:

Reviewer 2: ```suggestion
[APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst). The process is:
```

> requires the comment period. The process is:

Reviewer: I don't see anything in APE0 about maintainers, so it's not clear to me what the "which" in "which also requires the comment period" refers to.

Author: The APE0 part is about *all* the roles, not just maintainer roles.  But agreed this is confusingly worded, so I'll accept adrian's suggestion making this unnecessary!

> 1. Either the Coordination Committee or any community member including the 

Reviewer 3: Would it better here to elaborate what you mean by "any community member"? I assume it is anyone broadly working with Astropy, but I am not exactly sure.

Author: Yes, but in the latest commit I ended up removing this section.  But we should make a separate document about the nomination process to answer this question better, @aaryapatil !

   nominee may nominate someone for a role via private or public email,
   Slack message, in person request that is then recorded in meeting notes, etc.
2. A Coordination Committee member then sends a message to the person
   nominated, making sure they understand the role, its responsibilities, and
   confirming that the nominee agrees. Suggested text of that message is 
   [here](https://github.com/astropy/astropy-project/blob/main/messages/maintainer_access.md).

> 3. If the nominee does not accept, the process stopps here.

Reviewer 2: ```suggestion
3. If the nominee does not accept, the process stops here.
```

> 4. If the nominee does accept, a message goes out to the community, starting

Reviewer 2: ```suggestion
4. If the nominee does accept, a message is sent to the community, starting
```

   a two-week clock on the feedback period [as required by APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst#responsibilities-and-authority).
   The Coordination Committee member overseeing the process can make a PR at
   this time adding the member to the roles page as a place for public comment.
5. Once the two week period has elapsed, the coordination committee makes a

>    final decision based on any feedback. If the appointment is confirmed,
> 6. The nominee is added to the roles page (i.e., the PR created in step 4 can

Reviewer 2: ```suggestion
   final decision based on any feedback. 
6. If the appointment is confirmed, the nominee is added to the roles page (i.e., the PR created in step 4 can
```
